### PR TITLE
multilanguage support

### PR DESCRIPTION
**Pull request by lc-thomasberger**

### CODE REVIEW

File: grunt/tasks/translate/createLookupTable.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+      });
+    }
+    
+  });
+  
+};
+
+/*
+
+global.translate.lookupTables = {
```
Comment (brian-learningpool) [inline grunt/tasks/translate/createLookupTable.js]: Should this commented out block be removed?

File: grunt/tasks/translate/exportFile.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      next();
+    }
+    
+  });
+  
+};
+
+/*
+
+global.translate.exportTextData = [
```
Comment (brian-learningpool) [inline grunt/tasks/translate/exportFile.js]: More commented out code.

File: grunt/tasks/translate/parseSchema.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+      global.translate.schemaData.models[location].properties = _.extend(global.translate.schemaData.models[location].properties, properties);
+    }
+    
+  });
+  
+};
+
+/*
+
+global.translate.schemaData = {
```
Comment (brian-learningpool) [inline grunt/tasks/translate/parseSchema.js]: Maybe this should be removed too?

File: src/core/schema/contentobject.model.schema [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+    "displayTitle": {
+      "type":"string",
+      "required":false,
+      "default": "New Menu/Page Title",
+      "inputType":"DisplayTitle",
+      "validators": [],
+      "help": "When viewing a menu/page - this is the title that will be displayed on the menu/page",
+      "translatable": true
+    },
+    "body":{
```
Comment (brian-learningpool) [inline src/core/schema/contentobject.model.schema]: Do we need `pageBody` now too?

Comment (lc-thomasberger) [inline src/core/schema/contentobject.model.schema]: should be added in this PR https://github.com/adaptlearning/adapt_framework/pull/1271

File: grunt/tasks/translate/createLookupTable.js (hunk 1 above)
Comment (lc-thomasberger) [inline grunt/tasks/translate/createLookupTable.js]: fixed

**Comment by oliverfoster** [comment]
+1

**Pull request merged**